tests/arc/topt_no_cursor.nim

```diff
@@ -87,26 +87,26 @@ var lan_ip
 try:
   lan_ip = ""
   block label:
-    var line
     var a_cursor = txt
     var i = 0
     var L = len(a_cursor)
     block label_1:
       while true:
         if op(`<`(i, L)):
           break
-        var splitted
-        try:
-          line = a_cursor[i]
-          splitted = split(line, " ", -1)
-          if `==`(splitted[0], "opt"):
-            var :tmp = splitted[1]
-            `=copy`(lan_ip, :tmp)
-          echo([lan_ip])
-          echo([splitted[1]])
-          inc(i, 1)
-        finally:
-          `=destroy`(splitted)
+        block label_2:
+          var splitted
+          try:
+            var line = a_cursor[i]
+            splitted = split(line, " ", -1)
+            if `==`(splitted[0], "opt"):
+              var :tmp = splitted[1]
+              `=copy`(lan_ip, :tmp)
+            echo([lan_ip])
+            echo([splitted[1]])
+          finally:
+            `=destroy`(splitted)
+        inc(i, 1)
```

Comment:
The placement of the the `inc` outside of the try makes more sense, the isolation (clarified lifetimes) that the block provides allows for better intuitional understanding too!